codec-fix/README.md

```diff
@@ -49,5 +49,9 @@ E.g. original value: `100`, value after the removal of trailing zeroes: `1E+2`
 
 ## Release notes
 
++ 3.2.1
+  + Puts the BeginString(8) field during encode. The value to set is determined from the dictionary namespace.
```

Comment:
**Places** the BeginString(8) field during encode. The value to set **this field** is determined from the dictionary namespace.